src/lib/governance.test.js

```diff
@@ -3,18 +3,42 @@ import { expect, test } from 'vitest';
 import { outcomeMessage } from './governance';
 
 test('outcomeMessage', () => {
-  expect(outcomeMessage(undefined)).toBe('⏳ Vote Closes');
-  expect(outcomeMessage({})).toBe('???');
-  expect(outcomeMessage({ outcome: 'fail', reason: 'some reason' })).toBe(
-    '❌ some reason',
-  );
-  expect(outcomeMessage({ outcome: 'win', position: { noChange: {} } })).toBe(
-    '❌ Change Rejected',
-  );
-  expect(outcomeMessage({ outcome: 'win', position: { dontUpdate: {} } })).toBe(
-    '❌ Change Rejected',
-  );
-  expect(outcomeMessage({ outcome: 'win', position: { strings: [] } })).toBe(
-    '✅ Change Accepted',
-  );
+  expect(outcomeMessage(undefined)).toStrictEqual({
+    message: '⏳ Vote Closes',
+    color: 'bg-yellow-500 border border-yellow-100 bg-opacity-5',
```

Comment:
consider defining a class with a semantic name. I think you could even style this conditioned on the emoji with a selector like: `span[data-content^="❌"]`. I don't know how to mix that in with Tailwind